source/browser.ts

```diff
@@ -123,6 +123,10 @@ ipc.on('insert-emoji', async () => {
 	emojiElement.click();
 });
 
+ipc.on('insert-files', () => {
+	document.querySelector<HTMLElement>('._7oam input[type="file"]')!.click();
```

Comment:
I'd go with a single query:

```suggestion
	document.querySelector<HTMLElement>('._5vn8 + input[type="file"], ._7oam input[type="file"]')!.click();
```